xbmc/GUIInfoManager.cpp

```diff
@@ -9281,9 +9281,8 @@ void CGUIInfoManager::UpdateAVInfo()
 
       m_videoInfo = video;
       m_audioInfo = audio;
-
-      m_isPvrChannelPreview = g_PVRManager.IsChannelPreview();
     }
+    m_isPvrChannelPreview = g_PVRManager.IsChannelPreview();
```

Comment:
> How about setting IsChannelPreview to false in CGUIInfoManager::SetShowInfo if m_playerShowInfo is set to false?

Yeah that works - if CGUIInfoManager::ToggleShowInfo also resets m_isPvrChannelPreview. I've just learned that ToggleInfo is called when hitting "I". Earlier today I tried what you suggested (yeah had the same idea!), but only in SetShowInfo. That was obviously not sufficient...